frontend/templates/page/server.tsx

```diff
@@ -16,8 +16,9 @@ export const getServerSideProps: GetServerSideProps<PageTemplateProps> = async (
       };
    }
 
-   const slug = context.params?.slug;
+   const slug = context.params?.slug || [];
    invariant(Array.isArray(slug), 'page slug param is missing');
```

Comment:
This is kind of pointless now.

Reply:
I think it was kind of pointless before :-) if this wasn't here the next line would throw an exception.

Reply:
That said, it helps typescript to know that this is a `string[]` instead of `string`.